practices/mobile.md

```diff
@@ -1,56 +1,56 @@
+The Mobile Practice is distinct from the Web Practice at Artsy; although there is significant overlap between the
+two, there are some things that are necessarily very different when building mobile software that it warrants a
+mobile practice. Those things include:
 
 - Much of our code is written in Objective-C and Swift, which aren't commonly used within Artsy Engineering.
-- Testing is different from other Artsy systems (Front-End and Platform practices test on staging; we test in
-  betas).
-- Deploying is different from other Artsy systems (Front-End and Platform practices promote from staging to
-  production; we cut betas and submit to Apple for App Store approval).
-- Long-term maintenance is different (Front-End and Platform practices deploy code to hardware they control and can
-  patch easily; we deploy code to hardware our users control and can't guarantee upgrading at all, so we have to
-  anticipate how iOS code will continue to operate in production for _years_).
+- Testing is different from other Artsy systems (Web and Platform practices test on staging; we test in betas).
+- Deploying is different from other Artsy systems (Web and Platform practices promote from staging to production;
+  for iOS, we cut betas and submit to Apple for App Store approval).
+- Long-term maintenance is different (Wen and Platform practices deploy code to hardware they control and can patch
+  easily; we deploy code to hardware our users control and can't guarantee upgrading at all, so we have to
+  anticipate how mobile code will continue to operate in production for _years_).
 
-We do have a lot in common with the [Front-End Practice](./front-end.md):
+We do have a lot in common with the [Wen Practice](./web.md):
 
 - Large parts of our codebase are written in the [Artsy Omakase](./front-end.md#the-artsy-omakase).
 - Our React/Native apps share [a common design system](https://github.com/artsy/palette/).
 - We're all solving problems for Artsy.
 
-We also have a lot of non-iOS engineers contributing to our React Native app. Part of the Front-End iOS Practice,
-then, is to support Front-End engineers contributing to our iOS projects (mostly via React Native, but
+We also have a lot of non-mobile engineers contributing to our React Native app. Part of the Mobile Practice, then,
+is to support them contributing to our mobile projects (mostly via React Native, but
 [sometimes in native code](http://artsy.github.io/blog/2018/06/15/cocoapods-keys-react-native/)).
 
 ## Team Information
 
-Engineers who work on iOS code at artsy are included in the **@ios-front-enders** Slack group. They'll be
-automatically added to the #front-end-ios Slack channel. The Front-End iOS Practice meetings every two weeks;
-meeting notes are
-[available in Notion](https://www.notion.so/artsy/Front-End-iOS-ecc07763bfd04a848c74107dde3ec6dc).
+Engineers who work on mobile code at artsy are included in the **@ios-front-enders** Slack group. They'll be
```

Comment:
Hasn't the Slack group name changed?

Reply:
no actually, it still hasn't changed. We sent an email to IT to update it but we still haven't heard back from them. I will update it once it's done

Reply:
@MounirDhahri I can update it, what do you want to change it to?

Reply:
Thanks. we would like to update it to `@mobile-practice`

Reply:
I updated the PR to include the new group slack name, please feel free to merge it once you update it 🙏

Reply:
done